### PR TITLE
[Cisco FTD] Add parsing for addition message types

### PR DESCRIPTION
**Pull request by mjwolf**
## Proposed commit message

Add parsing for additional Cisco FTD message types.

The newly added message types are: 113042, 210007, 305006, 305013, 401004 and 500004

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

File: packages/cisco_ftd/data_stream/log/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -395,6 +395,25 @@ processors:
...
+        - "Non-HTTP connection from %{NOTCOLON:_temp_.cisco.source_interface}:%{IPORHOST:source.address}/%{NUMBER:source.port} to %{NOTCOLON:_temp_.cisco.destination_interface}:%{IPORHOST:destination.address}/%{NUMBER:destination.port} denied by redirect filter; only HTTP connections are supported for redirection."
+      pattern_definitions:
+        NOTCOLON: "[^:]*"
+        IPORHOST: "(?:%{IP}|%{HOSTNAME})"
+  - grok:
+      if: "ctx._temp_.cisco.message_id == '210007'"
+      field: "message"
+      description: "210007"
+      patterns:
+        - "LU allocate xlate failed for %{TYPE:_temp_.cisco.translation_type} %{WORD:network.protocol} translation from %{NOTCOLON:_temp_.cisco.source_interface}:%{IPORHOST:source.address}/%{NUMBER:source.port} \\(%{IPORHOST:_temp_.cisco.mapped_source_ip}/%{NUMBER:_temp_.cisco.mapped_source_port}\\) to %{NOTCOLON:_temp_.cisco.destination_interface}:%{IPORHOST:destination.address}/%{NUMBER:destination.port} \\(%{IPORHOST:_temp_.cisco.mapped_destination_ip}/%{NUMBER:_temp_.cisco.mapped_destination_port}\\)"
```
[the remark this answers is not in the document]
Comment (mjwolf) [reply packages/cisco_ftd/data_stream/log/elasticsearch/ingest_pipeline/default.yml]: I've added anchors to all the new grok patterns

File: packages/cisco_ftd/manifest.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 format_version: "3.0.3"
 name: cisco_ftd
 title: Cisco FTD
-version: "3.2.5"
+version: "3.2.6"
```
[the remark this answers is not in the document]
Comment (mjwolf) [reply packages/cisco_ftd/manifest.yml]: I've updated the version

File: packages/cisco_ftd/data_stream/log/elasticsearch/ingest_pipeline/default.yml [hunk 3]
```diff
@@ -395,6 +395,25 @@ processors:
         HOSTNAME: "\\b(?:[0-9A-Za-z][0-9A-Za-z_-]{0,62})(?:\\.(?:[0-9A-Za-z][0-9A-Za-z_-]{0,62}))*(\\.?|\\b)"
         IPORHOST: "(?:%{IP}|%{HOSTNAME})"
         CISCO_USER: ((LOCAL\\)?(%{HOSTNAME}\\)?%{USERNAME}(@%{HOSTNAME})?(, *%{NUMBER})?)
+  - grok:
+      if: "ctx._temp_.cisco.message_id == '113042'"
+      field: "message"
+      description: "113042"
```
[the remark this answers is not in the document]
Comment (mjwolf) [reply packages/cisco_ftd/data_stream/log/elasticsearch/ingest_pipeline/default.yml:401]: added tags to the new processors

File: packages/cisco_ftd/data_stream/log/elasticsearch/ingest_pipeline/default.yml [hunk 4]
```diff
@@ -474,6 +505,16 @@ processors:
...
+  - grok:
+      if: "ctx._temp_.cisco.message_id == '305013'"
+      field: "message"
+      description: "305013"
+      tag: "305013"
+      patterns:
+        - "^Asymmetric NAT rules matched for forward and reverse flows; Connection for %{NOTSPACE:network.transport} src %{NOTCOLON:_temp_.cisco.source_interface}:%{IPORHOST}/%{NUMBER:source.port}(\\(%{NOTSPACE:source.user.name}\\))? dst %{NOTCOLON:_temp_.cisco.destination_interface}:%{IPORHOST}/%{NUMBER:destination.port} denied due to NAT reverse path failure$"
+      pattern_definitions:
+        NOTCOLON: "[^:]*"
+        HOSTNAME: "\\b(?:[0-9A-Za-z][0-9A-Za-z\\-_]{0,62})(?:\\.(?:[0-9A-Za-z][0-9A-Za-z\\-_]{0,62}))*(\\.?|\\b)"
```
Comment (taylor-swanson) [inline packages/cisco_ftd/data_stream/log/elasticsearch/ingest_pipeline/default.yml:517]: This looks like an unused pattern.

**Review by taylor-swanson — APPROVED** [review]
LGTM

(Missed one item on the last look through).

File: packages/cisco_ftd/data_stream/log/elasticsearch/ingest_pipeline/default.yml (hunk 4 above)
Comment (mjwolf) [reply packages/cisco_ftd/data_stream/log/elasticsearch/ingest_pipeline/default.yml]: thanks for finding that, removed

**Pull request merged**